packages/cli/src/index.ts

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { generate } from './commands/generate';
+
+yargs(hideBin(process.argv))
+    .command(
+        'generate',
+        'Generate a schema from supplied database connection URL',
+        {
+            connectionString: {
+                alias: 'url',
+                default: 'postgres://postgres:postgres@localhost:5432/postgres',
+            },
+            outputDir: {
+                alias: 'out',
+                default: 'src',
+            },
+            defaultSchema: {
+                alias: 'default',
```

Comment:
In fact here's my rule of thumb for command/param aliases:

1. If the command/param is infrequently invoked (i.e. typed) by the user, prefer longer more descriptive names. 
2. If the command is often typed manually by the user, add one or two letter aliases. Example: `git status` => `git s`
3. Never add synonyms or more than one name to a command or param. If people find examples online this will eventually just lead to confusion.